### PR TITLE
Add messages in Uzbek

### PR DESCRIPTION
**Pull request by mrmuminov**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ✔️
| Breaks BC?    | ❌

### CODE REVIEW

File: messages/uz/yii-validator.php [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    'Must not be an IPv4 address.' => 'IPv4 boʻlmasligi kerak.',
+    'Must not be an IPv6 address.' => 'IPv6 boʻlmasligi kerak.',
+    'Contains wrong subnet mask.' => 'Notoʻgʻri quyi tarmoq(subnet) maskasi.',
+    'Must be an IP address with specified subnet.' => 'Quyi tarmoq(subnet)ga ega IP manzil boʻlishi kerak.',
+    'Must not be a subnet.' => 'Quyi tarmoq(subnet) boʻlmasligi kerak.',
+    'Is not in the allowed range.' => 'Qiymat ruxsat berilgan oraliqda emas.',
+    'The value must be "{true}".' => 'Qiymat «{true}» boʻlishi kerak.',
+    'The value is not JSON.' => 'Qiymat JSON holatida emas.',
+    'Nested rule without rules can be used for objects only.' => 'Qoidalarga ega boʻlmagan Nested qoidasi faqat obyektlarga ishlatilishi mumkin.',
+    'An object data set data can only have an array or an object type.' => 'Maʻlumotlar toʻplami qiymatlari massiv yoki obyekt boʻlishi kerak.',
```
Comment (rustamwin) [inline messages/uz/yii-validator.php:41]:
```suggestion
    'An object data set data can only have an array or an object type.' => 'Obyekt ma\'lumotlari massiv yoki obyekt boʻlishi kerak.',
```

File: messages/uz/yii-validator.php [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    'The value must have an array or an object type.' => 'Qiymat massiv yoki obyekt boʻlishi kerak.',
+    'Property "{path}" is not found.' => '{path} xususiyati topilmadi.',
+    'The allowed types are integer, float and string.' => 'Ruxsat berilgan turlar: integer, float va string.',
+    'Value must be an integer.' => 'Qiymat butun son boʻlishi kerak.',
+    'Value must be a number.' => 'Qiymat haqiqiy raqam boʻlishi kerak.',
```
Comment (rustamwin) [inline messages/uz/yii-validator.php:46]:
```suggestion
    'Value must be a number.' => 'Qiymat raqam boʻlishi kerak.',
```

File: messages/uz/yii-validator.php [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+    'Value must be no less than {min}.' => 'Qiymat {min} dan kichik boʻlmasligi kerak.',
+    'Value must be no greater than {max}.' => 'Qiymat {max} dan katta boʻlmasligi kerak.',
+    'Value should be string.' => 'Qiymat satr boʻlishi kerak',
+    'Value is invalid.' => 'Qiymat notoʻgʻri.',
+    'Value cannot be blank.' => 'Qiymat boʻsh boʻlishi mumkin emas.',
+    'Value not passed.' => 'Qiymat oʻtmadi.',
+    'Value must be iterable.' => 'Qiymat iteratsiyalangan boʻlishi kerak',
```
Comment (rustamwin) [inline messages/uz/yii-validator.php:53]:
```suggestion
    'Value must be iterable.' => 'Qiymat takrorlanadigan boʻlishi kerak',
```

File: messages/uz/yii-validator.php [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+    'This value is not a subset of acceptable values.' => 'Bu qiymat qabul qilinadigan qiymatlar to\'plami emas.',
```
Comment (rustamwin) [inline messages/uz/yii-validator.php:54]:
```suggestion
    'This value is not a subset of acceptable values.' => 'Bu qiymat ruxsat etilgan qiymatlarning quyi toʻplami emas.',
```

File: messages/uz/yii-validator.php [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+    'This value is not a valid URL.' => 'Qiymat to\'g\'ri havola emas.',
```
Comment (rustamwin) [inline messages/uz/yii-validator.php:55]:
```suggestion
    'This value is not a valid URL.' => 'Qiymat yaroqli havola emas.',
```

**Review by rustamwin — CHANGES_REQUESTED** [review]

**Review by mrmuminov — COMMENTED** [review]
Corrections in translation, spelling and plural elements

File: messages/uz/yii-validator.php [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+return [
+    'The data must have at least "{min}" filled attributes.' => 'Kamida {min} ta atributlar toʻldirilgan boʻlishi kerak.',
+    'Value must be an array or an object.' => 'Qiymat massiv yoki obyekt boʻlishi kerak.',
+    'The allowed types are integer, float, string, boolean. {type} given.' => 'Ruxsat berilgan turlar: integer, float, string, boolean va null. Bu qiymat turi {type}',
+    'Value must be either "{true}" or "{false}".' => 'Qiymat "{true}" yoki "{false}" boʻlishi kerak.',
+    'The allowed types are integer, float, string, boolean and null.' => 'Ruxsat berilgan turlar: integer, float, string, boolean va null.',
+    'The attribute value returned from a custom data set must have a scalar type.' => 'Maʼlumotlar toʻplamidan olingan qiymat skalyar turdagi boʻlishi kerak.',
+    'Value must be equal to "{targetValueOrAttribute}".' => 'Qiymat "{targetValueOrAttribute}"ga teng boʻlishi kerak.',
+    'Value must not be equal to "{targetValueOrAttribute}".' => 'Qiymat "{targetValueOrAttribute}"ga teng boʻlmasligi kerak.',
+    'Value must be greater than "{targetValueOrAttribute}".' => 'Qiymat {targetValueOrAttribute} dan katta boʻlishi kerak.',
```
Comment (rustamwin) [inline messages/uz/yii-validator.php:14]:
```suggestion
    'Value must be greater than "{targetValueOrAttribute}".' => 'Qiymat "{targetValueOrAttribute}"dan katta boʻlishi kerak.',
```

File: messages/uz/yii-validator.php [hunk 7]
```diff
@@ -0,0 +1,56 @@
...
+    'Value must be greater than or equal to "{targetValueOrAttribute}".' => 'Qiymat {targetValueOrAttribute} dan katta yoki teng boʻlishi kerak.',
```
Comment (rustamwin) [inline messages/uz/yii-validator.php:15]:
```suggestion
    'Value must be greater than or equal to "{targetValueOrAttribute}".' => 'Qiymat "{targetValueOrAttribute}"dan katta yoki teng boʻlishi kerak.',
```

**Review by rustamwin — CHANGES_REQUESTED** [review]

**Review by rustamwin — APPROVED** [review]

**Pull request merged**